tests/apps/portal/test_models.py

```diff
@@ -105,3 +108,34 @@ def test_generate_css__deletes_old_files(self):
 
         self.assertFalse(path.exists(self.full_path))
         self.assertTrue(path.exists(path.join(settings.MEDIA_ROOT, 'linkmap', self.css_file)))
+
+
+class TestPrivateMessageEntry(TestCase):
+
+    def setUp(self):
+        super().setUp()
+
+        user = User.objects.register_user('testing', 'example@example.com',
+                                               'pwd', False)
+        user.groups.add(Group.objects.get(name=settings.INYOKA_TEAM_GROUP_NAME))
+        self.other_user = User.objects.register_user(
+            'other_user',
+            'example2@example.com',
+            'pwd', False)
+        pm = PrivateMessage(author=user, subject="Expired message", pub_date=datetime.now() -
+            timedelta(days=settings.PRIVATE_MESSAGE_INBOX_SENT_DURATION))
+        pm.send([self.other_user])
+
+        self.privmsgentry = PrivateMessageEntry.objects.get(message=pm, user=self.other_user)
+
+    def test_delete_messages(self):
+        self.assertEqual(self.privmsgentry.message.subject, 'Expired message')
+        PrivateMessageEntry.clean_private_message_folders()
+        self.assertFalse(PrivateMessageEntry.objects.filter(folder=1).exists())
+        self.assertTrue(PrivateMessageEntry.objects.filter(folder=0).exists())
```

Comment:
for clarity i would add a short comment that it's a team member (and that's the reason why the message entry is still here)